app/views/achievements/_grid.html.haml

```diff
@@ -1,11 +1,13 @@
 - player ||= nil
 .row.achievement_grid
   - achievements.each do |achievement|
-    .col-md-3.achievement
+    .col-md-4.achievement
       .panel.panel-default{class: achievement_class(achievement, player)}
-        .panel-heading=achievement.title
+        .panel-heading
+          = achievement.title
+          .points.pull-right{title: t(:achievement_points)}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace